includes/validation/class-amp-validated-url-post-type.php

```diff
@@ -2052,15 +2052,17 @@ static function ( $a, $b ) use ( $stylesheets ) {
 					<?php esc_html_e( 'Total CSS size prior to minification:', 'amp' ); ?>
 				</th>
 				<td>
-					<?php echo esc_html( number_format_i18n( $included_original_size + $excluded_original_size ) ); ?><small>B</small>
+					<?php echo esc_html( number_format_i18n( $included_original_size + $excluded_original_size ) ); ?>
+					<abbr title="<?php esc_attr_e( 'bytes', 'amp' ); ?>"><?php echo esc_attr_x( 'B', 'abbreviation for bytes', 'amp' ); ?></abbr>
```

Comment:
@swissspidy Is this the right usage of translation context?

Reply:
I think it's OK.

One could use `file size` or something as context, and `abbreviation for bytes` as translator comment, but that's mostly nitpicking in this case.